File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/iget-all-participants_item.ts




export interface IGetAllParticipants_Item {
  
	PersonID: number;
	CountryID?: number;
	FirstMiddleNames: string;
	Gender: string;
	AgencyName: string;
	RankName: string;
	JobTitle: string;
	LastVettingTypeCode: string;
	LastVettingStatusCode: string;
	LastVettingStatusDate?: Date;
	ParticipantType: string;
	Distinction: string;
	UnitName: string;
	VettingValidEndDate?: Date;
	TrainingDate?: Date;
	DOB?: Date;
	UnitNameEnglish: string;
	UnitAcronym: string;
	AgencyNameEnglish: string;
	NationalID: string;

}

File: GTTS/INL.PersonService.Models/IGetAllParticipants_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.PersonService.Models
{
    public interface IGetAllParticipants_Item
    {
        long PersonID { get; set; }
        int? CountryID { get; set; }
        string FirstMiddleNames { get; set; }
        string Gender { get; set; }
        string AgencyName { get; set; }
        string RankName { get; set; }
        string JobTitle { get; set; }
        string LastVettingTypeCode { get; set; }
        string LastVettingStatusCode { get; set; }
        DateTime? LastVettingStatusDate { get; set; }
        string ParticipantType { get; set; }
        string Distinction { get; set; }
        string UnitName { get; set; }
        DateTime? VettingValidEndDate { get; set; }
        DateTime? TrainingDate { get; set; }
        DateTime? DOB { get; set; }
        string UnitNameEnglish { get; set; }
        string UnitAcronym { get; set; }
        string AgencyNameEnglish { get; set; }
        string NationalID { get; set; }
    }
}
